docs/notebooks/inequality_constraints.pct.py

```diff
@@ -148,16 +153,16 @@ def create_bo_model(data):
 
 # %%
 constraint_data = result.datasets[CONSTRAINT]
-new_data = (
-    constraint_data.query_points[-num_steps:], constraint_data.observations[-num_steps:]
-)
+new_query_points = constraint_data.query_points[-num_steps:]
+new_observations = constraint_data.observations[-num_steps:]
+new_data = (new_query_points, new_observations)
```

Comment:
Apologies, this is not a black change. But I couldn't leave as is after @joelberkeley remarks :)